src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user1 = new User(email, password);
```

Comment:
Do you really need to create an additional user object, or maybe it would be enough just to compare fields of the existing one with password and email?